src/tvu.rs

```diff
@@ -34,7 +33,7 @@ use std::sync::{Arc, RwLock};
 use std::thread;
 
 pub struct TvuRotationInfo {
-    pub bank: Arc<Bank>,     // Bank to use
```

Comment:
you wouldn't rather just take a bank and a slot?

Reply:
@rob-solana block tree doesn't have any data for the next bank yet.